meson.build

```diff
@@ -57,6 +58,12 @@ if shellcheck.found()
   test('shellcheck toolbox (deprecated)', shellcheck, args: [toolbox_sh])
 endif
 
+if codespell.found()
+  test('codespell toolbox', codespell, args: [meson.current_source_dir() / 'src' , 
+  meson.current_source_dir() / 'cmd', meson.current_source_dir() / 'profile.d', meson.current_source_dir() / 'doc', 
```

Comment:
Is `meson.current_source_dir() / 'cmd'` correct?  We don't have a `cmd` sub-directory under the top-level directory.  We have it under `src`.

Reply:
As codespell works recursively we don't need to add 'cmd' because it is inside 'src'.